config/prompts_sys.py

```
@@ -398,4 +398,26 @@ def get_proactive_chat_prompt(kind: str, lang: str = 'zh') -> str:
+
+
+# ============ 普通对话截断配置 ============
+
+# 普通对话改写提示词（用于精简过长回复）
+normal_chat_rewrite_prompt = """你是一个文本精简专家。请将以下AI回复进行精简和清洁。
+
+======以下为原始回复======
+{raw_output}
+======以上为原始回复======
+
+请按照以下规则处理：
```

Ruff (0.14.14)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)